qa/rpc-tests/show_help.py

```diff
@@ -0,0 +1,504 @@
+       blockchain through -rescan on startup (1 = keep tx meta data e.g.
+       account owner and payment request information, 2 = drop tx meta data)
+
+  -walletrequirebackup=<bool>
+       By default, the wallet will not allow generation of new spending keys &
+       addresses from the mnemonic seed until the backup of that seed has been
+       confirmed with the `zcashd-wallet-tool` utility. A user may start zcashd
+       with `-walletrequirebackup=false` to allow generation of spending keys
+       even if the backup has not yet been confirmed.
+
+ZeroMQ notification options:
+
+  -zmqpubhashblock=<address>
+       Enable publish hash block in <address>
+
+  -zmqpubhashtx=<address>
+       Enable publish hash transaction in <address>
+
+  -zmqpubrawblock=<address>
+       Enable publish raw block in <address>
+
+  -zmqpubrawtx=<address>
+       Enable publish raw transaction in <address>
+
+Monitoring options:
+
+  -metricsallowip=<ip>
+       Allow metrics connections from specified source. Valid for <ip> are a
+       single IP (e.g. 1.2.3.4), a network/netmask (e.g. 1.2.3.4/255.255.255.0)
+       or a network/CIDR (e.g. 1.2.3.4/24). This option can be specified
+       multiple times. (default: only localhost)
+
+  -metricsbind=<addr>
+       Bind to given address to listen for metrics connections. (default: bind
+       to all interfaces)
+
+  -prometheusport=<port>
+       Expose node metrics in the Prometheus exposition format. An HTTP
+       listener will be started on <port>, which responds to GET requests on
+       any request path. Use -metricsallowip and -metricsbind to control
+       access.
+
+Debugging/Testing options:
+
+  -debug=<category>
+       Output debugging information (default: 0, supplying <category> is
+       optional). If <category> is not supplied or if <category> = 1, output
+       all debugging information. <category> can be: addrman, alert, bench,
+       coindb, db, estimatefee, http, libevent, lock, mempool, mempoolrej, net,
+       partitioncheck, pow, proxy, prune, rand, receiveunsafe, reindex, rpc,
+       selectcoins, tor, zmq, zrpc, zrpcunsafe (implies zrpc). For multiple
+       specific categories use -debug=<category> multiple times.
+
+  -experimentalfeatures
+       Enable use of experimental features
+
+  -help-debug
+       Show all debugging options (usage: --help -help-debug)
+
+  -logips
+       Include IP addresses in debug output (default: 0)
+
+  -logtimestamps
+       Prepend debug output with timestamp (default: 1)
+
+  -minrelaytxfee=<amt>
+       Fees (in ZEC/kB) smaller than this are considered zero fee for relaying,
+       mining and transaction creation (default: 0.000001)
+
+  -maxtxfee=<amt>
+       Maximum total fees (in ZEC) to use in a single wallet transaction or raw
+       transaction; setting this too low may abort large transactions (default:
+       0.10)
+
+  -printtoconsole
+Send trace/debug info to console instead of the debug log
```

Comment:
```suggestion
       Send trace/debug info to console instead of the debug log
```
Argh! There’s some weird copy behavior in GH where it doesn’t seem to copy the leading whitespace, so the suggestion to this introduced a new issue 😕